### PR TITLE
Spanish translation updated

### PR DESCRIPTION
**Pull request by macomarivas**
Hi! updated the Spanish translation

### CODE REVIEW

**Review by Nordsoft91 — COMMENTED** [review]
I see indents are broken in spanish.json
can you please replace spaces with tabs everywhere? (it's ok to do in separate PR, if you'd like to)

File: launcher/translation/spanish.ts [hunk 1]
```diff
@@ -456,17 +487,17 @@
     <message>
         <location filename="../settingsView/csettingsview_moc.ui" line="552"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translationDefecto</translation>
```
Comment (Nordsoft91) [inline launcher/translation/spanish.ts:490]:
```suggestion
        <translation>Defecto</translation>
```

**Comment by IvanSavenko** [comment]
@macomarivas 
FYI - you can use Qt Linguist to edit .ts file instead of doing it manually:
Can be found for example here: https://github.com/lelegard/qtlinguist-installers/releases

**Comment by macomarivas** [comment]
thanks @IvanSavenko, changes updated

**Pull request merged**